website/content/en/docs/olm-integration/legacy/quickstart-bundle.md

```diff
@@ -0,0 +1,149 @@
+catalogsources.operators.coreos.com                          CustomResourceDefinition    Installed
+subscriptions.operators.coreos.com                           CustomResourceDefinition    Installed
+...
+```
+
+All resources listed should have status `Installed`.
+
+If OLM is not already installed, go ahead and install the latest version:
+
+```console
+$ operator-sdk olm install
+INFO[0000] Fetching CRDs for version "latest"
+INFO[0001] Fetching resources for version "latest"
+INFO[0007] Creating CRDs and resources
+INFO[0007]   Creating CustomResourceDefinition "clusterserviceversions.operators.coreos.com"
+INFO[0007]   Creating CustomResourceDefinition "installplans.operators.coreos.com"
+INFO[0007]   Creating CustomResourceDefinition "subscriptions.operators.coreos.com"
+...
+NAME                                            NAMESPACE    KIND                        STATUS
+clusterserviceversions.operators.coreos.com                  CustomResourceDefinition    Installed
+installplans.operators.coreos.com                            CustomResourceDefinition    Installed
+subscriptions.operators.coreos.com                           CustomResourceDefinition    Installed
+catalogsources.operators.coreos.com                          CustomResourceDefinition    Installed
+...
+```
+
+**Note:** By default, `olm status` and `olm uninstall` auto-detect the OLM version installed in your cluster.
+This can fail if the installation is broken in some way, so the version of OLM can be overridden using the
+`--version` flag provided with these commands.
+
+## Creating a bundle
+
+_If working with package manifests, see the [package manifests quickstart][quickstart-package-manifests]._
+
+We will now create bundle manifests and metadata by running `generate bundle` in the root of the memcached-operator project.
+
+```sh
+$ operator-sdk generate bundle --version 0.0.1
+```
+
+A bundle manifests directory `deploy/olm-catalog/memcached-operator/manifests` containing a CSV and all CRDs
+in `deploy/crds`, a bundle [metadata][bundle-metadata] directory `deploy/olm-catalog/memcached-operator/metadata`,
+and a [Dockerfile][bundle-dockerfile] `bundle.Dockerfile` have been created in the Operator project.
+
+These files can be statically validated by `bundle validate` to ensure the on-disk bundle representation is correct:
+
+```console
+$ operator-sdk bundle validate ./deploy/olm-catalog/memcached-operator
+INFO[0000] Found annotations file                        bundle-dir=deploy/olm-catalog/memcached-operator container-tool=docker
+INFO[0000] Could not find optional dependencies file     bundle-dir=deploy/olm-catalog/memcached-operator container-tool=docker
+INFO[0000] All validation tests have completed successfully
+```
+
+## Deploying an Operator with OLM
+
+At this point in development we've generated all files necessary to build the memcached-operator bundle.
+Now we're ready to test and deploy the Operator with OLM.
+
+### Testing bundles
+
+Coming soon.
```

Comment:
Nothing is coming soon for legacy operator projects, right?

Reply:
Fair point. I'll add a note that you must migrate to the new CLI to get this feature.